llm-server/src/model/llama-model-provider.ts

```diff
@@ -68,4 +68,36 @@ export class LlamaModelProvider extends ModelProvider {
       res.end();
     }
   }
+
+  async getModelTagsResponse(res: Response): Promise<void> {
```

Comment:
for this part, assume a config or param from user? read the ollama endpoint get the result